webapp/components/Comment.vue

```diff
@@ -14,6 +14,7 @@
         placement="bottom-end"
         resource-type="comment"
         :resource="comment"
+        :callbacks="{ confirm: deleteCommentCallback, cancel: null }"
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-06-03T12:56:03Z" title="Monday, June 3rd 2019, 2:56:03 pm +02:00">Jun 3, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -14,6 +14,7 @@
         placement="bottom-end"
         resource-type="comment"
         :resource="comment"
+        :callbacks="{ confirm: deleteCommentCallback, cancel: null }"
```


Ideally, you would give each callback a separate property:

```suggestion
        :confirm="deleteCommentCallback"
        // :cancel="null" just omit
```